desktop/apps/auction2/components/sidebar/index.js

```diff
@@ -0,0 +1,17 @@
+import classNames from 'classnames'
+import { default as React, PropTypes } from 'react';
+import ArtistFilterContainer from '../artist_filter/index'
+import MediumFilterContainer from '../medium_filter/index'
+import RangeSliderContainer from '../range_slider/index'
+
+export default function Sidebar() {
+  return (
+    <div className='auction2-artworks-sidebar'>
+      <div className='auction2-artworks-sidebar__artist-filter'>
+        <RangeSliderContainer />
+        <MediumFilterContainer />
+        <ArtistFilterContainer />
```

Comment:
Why not keep it nice and literal? 

```javascript
<div className='auction2-artworks-sidebar__artist-filter'>
  <RangeSlider />
  <MediumFilter />
  <ArtistsFilter /> 
</div>
```

Also, don't need to explicitly import things that have an index; the folder name assumes there's an `index` file. 

`import RangeSliderContainer from '../range_slider/'`